act/scio/api.py

```diff
@@ -264,7 +264,7 @@ def download(
 def download_json(
     id: SHA256Regex,
     args: argparse.Namespace = Depends(parse_args),
-) -> Union[Response, Dict[Text, Text]]:
+) -> Dict[Text, Any]:
```

Comment:
This failed in newer versions of FastAPI, since it checked the response code. Only dictionaries are returned here so the old return value was wrong